variables.tf

```diff
@@ -4,6 +4,12 @@ variable "model" {
   default     = "identity-platform"
 }
 
+variable "deploy_extra_charms" {
+  type        = bool
+  default     = false
+  description = "Whether to deploy Identity Platform extensions: Admin UI, External IdP integrator, OpenFGA"
+}
```

Comment:
I think this should be removed, I think the idea is that you can create a new tf plan and combine these 2 modules there.